Strings/Scan.jsonc

```diff
@@ -106,10 +106,10 @@
 
 	"モバイルSuica/PASMO に転送済みのカードや有効期限が切れたカードでも残高が表示されますが、実際に使用することはできません。": {
 		"ja": "モバイルSuica/PASMO に転送済みのカードや有効期限が切れたカードでも残高が表示されますが、実際に使用することはできません。",
-		"en": null,
-		"zh-Hans": null,
-		"zh-Hant": null,
-		"zh-HK": null
+		"en": "The balance will still be displayed even if the card has been transferred to Mobile Suica/PASMO, or if the card has already expired, but it cannot be used.",
```

Comment:
The sentence structure here is a bit strange, perhaps this would work better;

> If the card has been transferred to Mobile Suica/PASMO or the card has expired, the balance and history will still be displayed up until that point, but will remain inoperable.

Reply:
Thanks, that sounds more natural, though I think "until that point" should be placed in front of "the balance and history" for clarity ("displayed up until that point" might be misinterpreted as "the balance and history will disappear when the card is transferred/expires"). How about this:

> If the card has been transferred to Mobile Suica/PASMO or has already expired, the balance and history up until that point will still be displayed, but the card will remain unusable.

Reply:
I think from that angle, we should remove "still" from "will still be displayed" as it's kinda reinforcing what we're changing it to not be like.

> the balance and history up until that point will be displayed, but the card will remain unusable.

Great!

Reply:
Hmm, I just checked the Japanese string, it appears that "history" isn't mentioned (just "balance"). Am I missing something?

Reply:
Uh let's ask @treastrain on this one.

Reply:
oh, I'm sorry. In that situation, we'll see both the balance and the usage history at that point in time, just like a card that hasn't been transferred to Mobile Suica/PASMO.

However, I want to tell the user that "the card's balance can no longer be used" here.

Reply:
I think our translation accurately represents that, but we can specify it further; How's this?

> If the card has been transferred to Mobile Suica/PASMO or has expired, the history and balance up until that point will be displayed, but the card and its balance will remain unusable.

Reply:
> but the card and its balance will remain unusable.

Do we really need to mention the balance here? I think "the card will remain unusable" already implies that the balance is also unusable, since if the card itself is not usable, there's no way the balance could be accessed anyway.